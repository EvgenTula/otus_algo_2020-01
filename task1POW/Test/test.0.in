1,000000001
100000