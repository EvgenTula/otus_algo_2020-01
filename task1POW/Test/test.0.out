1,000100005000116665833199997639